### DISCUSSION TITLE
Vite doesn't update included js files when changed

### DISCUSSION THREAD

**Discussion by kirschkern** [quasarframework/quasar#13730] [General - Components / Directives / etc] [answered]
In my Quasary CLI project I include my own js files like this:
`import Task from "src/js/task.js"`

When I change something in this file (task.js) and save it, my changes are not visible or functional with hot reload. However I see this in the console:
`[vite] hot updated: /src/components/kk-phase-tasks.vue`

Even after reloading the page my changes are not build in.

Only after restarting with quasar dev my changes are reflected.

Do I have to define folders or files that vite should update?
What am I missing? 

Thanks for any help
Daniel

quasar v2.7.1
@quasar/app-vite v1.0.1

**Comment by alexrififi** [comment]
Everything works out of the box.
Show vite config.

In such cases, I usually install a new project, do not change any configs and try it there. If it works - the problem is in my configs

**Reply by kirschkern** [reply]
There is no vite.config.js in my quasar project.
Where do I find it?

**Comment by kirschkern** [comment, marked as answer]
The reason for Vite not updating my js files with hot reload was an uppercase letter in my import statement. My import was actually:

`import Task from "src/js/Task.js"`

but the filename is `task.js` (lowercase).

While the file will be included when the server starts, it will not be reloaded when the file changes.
